lib/percy/capybara.rb

```diff
@@ -54,8 +54,10 @@ def self.disable!
     end
 
     def self.use_loader(loader, options = {})
-      capybara_client.loader = loader
-      capybara_client.loader_options = options
+      client = capybara_client(loader: loader)
```

Comment:
@fotinakis at very minimum we can't use keyword arguments, like in your example, to maintain support for Ruby < 2.0.